### PR TITLE
feat(top up page): init pages to test checkout session and ar top up

### PR DESCRIPTION
**Pull request by fedellen**

### CODE REVIEW

File: src/pages/TopUpPage.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            <a href="https://www.youtube.com/watch?v=51Gpg3-GFZw">
+              Get a Wallet
+            </a>
+          </li>
+          <li>
+            <a href="https://www.youtube.com/watch?v=51Gpg3-GFZw">
+              HOW TO: Upload with ArDrive Turbo
+            </a>
+          </li>
+        </ul>
```
Comment (fedellen) [inline src/pages/TopUpPage.tsx]: todo: component

File: src/Router.tsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+import { TopUpARPage } from "./pages/ArTopUpPage";
 
 export function Router() {
   return (
     <BrowserRouter basename={`${import.meta.env.BASE_URL}`}>
       <Routes>
         <Route path="/gift" element={<GiftPage />} />
         <Route path="/redeem" element={<RedeemPage />} />
+        <Route path="/top-up" element={<TopUpWalletAddressPage />} />
+        <Route path="/ar-top-up" element={<TopUpARPage />} />
```
Comment (fedellen) [inline src/Router.tsx]: idea: /top-up/arweave /top-up/fiat. with /top-up linking to both pages? or just fiat

Comment (dtfiedler) [reply src/Router.tsx]: /top-up/crypto with a dropdown on the page of what cyrpto component you want to top up with

File: src/pages/CryptoTopUpPage.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  const [sending, setSending] = useState<boolean>(false);
+
+  useEffect(() => {
+    getArconnect().then(async () => {
+      const arconnect_signer = new ArconnectSigner(window.arweaveWallet);
+      if (arconnect_signer.publicKey === undefined) {
+        await arconnect_signer.setPublicKey();
+      }
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```
Comment (dtfiedler) [inline src/pages/CryptoTopUpPage.tsx]: do we know why we need this?

Comment (fedellen) [reply src/pages/CryptoTopUpPage.tsx]: we no longer need it (:

Comment (fedellen) [reply src/pages/CryptoTopUpPage.tsx]: [e2a83ac](https://github.com/ardriveapp/turbo-app/pull/13/commits/e2a83ace674b13867e3561cf3ee6df9ff586cb57)

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**